packages/core-data/README.md

```diff
@@ -40,6 +40,60 @@ function MyAuthorsListBase() {
 }
 ```
 
+## What's an entity?
+
+An entity represents a WordPress REST API endpoint. Each item within the entity is called entity record. Available entities are defined in `rootEntitiesConfig` at ./src/entities.js.
+
+What follows is a description of some of the properties of `rootEntitiesConfig`.
+
+## baseURL
+
+- Type: string.
+- Example: `'/wp/v2/users'`.
+
+This property maps the entity to a given endpoint, representing its URL as defined in the [REST API handbook](https://developer.wordpress.org/rest-api/reference/#rest-api-developer-endpoint-reference).
+
+## baseURLParams
+
+- Type: `object`.
+- Example: `{ context: 'edit' }`.
+
+Additional parameters to the request, if the endpoint supports it. The additional arguments available to the request are listed under arguments section. As an example, see the user entity and the [users endpoint](https://developer.wordpress.org/rest-api/reference/users/#list-users). By providing `{ context: 'edit' }` the server response will include all the fields in the [user schema](https://developer.wordpress.org/rest-api/reference/users/#schema) that belong to the `edit` context.
+
+## key
+
+- Type: `string`.
+- Example: `'slug'`.
+
+The endpoint response may be in different formats. It can be a simple object, which maps to a single entity record. This is the case of the site entity (settings endpoint):
```

Comment:
```suggestion
The endpoint response may be in different formats. It can be a simple object, which maps to a single entity record. This is the case for the site entity (settings endpoint):
```